config/registry/default_schema.yaml

```diff
@@ -0,0 +1,693 @@
+        associated with a particular SkyMap.  The SQL representation is a table with
+        the spatial region for each tract.
+      columns:
+      - !Column
+        name: skymap
+        type: string
+        primary_key: true
+        nullable: false
+      - !Column &tract
+        name: tract
+        type: int
+        primary_key: true
+        nullable: false
+      - !Column
+        name: region
+        type: blob
+      constraints:
+      - !ForeignKeyConstraint
+        src: skymap
+        tgt: SkyMap.skymap
+    
+    - !DataUnit
+      name: Patch
+      dependencies:
+      - SkyMap
+      - Tract
+      doc: >
+        A rectangular region within a tract.  The SQL representation is a table with
+        the spatial region for each tract+patch combination and the patch's location
+        within the tract's grid.
+      columns:
+      - !Column
+        name: skymap
+        type: string
+        primary_key: true
+        nullable: false
+      - !Column
+        name: tract
+        type: int
+        primary_key: true
+        nullable: false
+      - !Column &patch
+        name: patch
+        type: int
+        primary_key: true
+        nullable: false
+      - !Column
+        name: cell_x
+        type: int
+        nullable: false
+      - !Column
+        name: cell_y
+        type: int
+        nullable: false
+      - !Column
+        name: region
+        type: blob
+        nullable: false
+      constraints:
+      - !ForeignKeyConstraint
+        src: skymap
+        tgt: SkyMap.skymap
+      - !ForeignKeyConstraint
+        src:
+        - skymap
+        - tract
+        tgt:
+        - Tract.skymap
+        - Tract.tract
+    
+    - !DataUnit
+      name: Label
+      doc: >
+        An arbitrary string value.  There is no SQL representation or constraint on
+        the values a Label can take.
+      columns:
+      - !Column &label
+        name: label
+        type: string
+  
+  tables:
+    - !Table
+      name: Datasets
+      columns:
+      - !Column
+        name: dataset_id
+        type: int
+        primary_key: true
+        nullable: false
+        doc: >
+          Autoincrement
+      - !Column
+        name: registry_id
+        type: int
+        primary_key: true
+        nullable: false
+        doc: >
+          A different number for each dataset_id incrementer (usually each different
+          Registry).
+      - !Column
+        name: dataset_type_name
+        type: string
+        nullable: false
+      - !Column
+        name: uri
+        type: string
+      - !Column
+        name: run_id
+        type: int
+        nullable: false
+        doc: >
+          Link to the Run table.
+      - !Column
+        name: quantum_id
+        type: int
+        doc: >
+          Link to the Quantum table (only present for SuperTask-generated Datasets).
+      # DataUnit values
+      - !Column
+        <<: *camera
```

Comment:
Are these going to inherit the `primary_key` attributes these column definitions have in the `DataUnits` section?

Reply:
Yes, unless overridden in the same way as `nullable` is.